<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menú Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>521</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Componente Práctico - Matemáticas Discretas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Sebastián Andres Zárate Sanmiguel 448669&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Puede utilizar el menú de la parte superior izquierda &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;para lanzar las tres aplicaciones realizadas para esta entrega &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;o también puede aprovechar la interfaz a continuación.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>80</y>
      <width>631</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccione una aplicación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Calculadora números primos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Calculadora secuencia Fibonacci</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Calculadora de bases numéricas</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lanzar Aplicación</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Aplicaciones</string>
    </property>
    <addaction name="Archivo_AbrirVentana1"/>
    <addaction name="Archivo_AbrirVentana2"/>
    <addaction name="actionAbrir_calculadora_de_bases"/>
    <addaction name="Archivo_Salir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Archivo_AbrirVentana1">
   <property name="text">
    <string>Abrir calculadora primos</string>
   </property>
  </action>
  <action name="Archivo_AbrirVentana2">
   <property name="text">
    <string>Abrir calculadora Fibonacci</string>
   </property>
  </action>
  <action name="Archivo_Salir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAbrir_calculadora_de_bases">
   <property name="text">
    <string>Abrir calculadora de bases</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
